db/schema.rb

```diff
@@ -342,6 +343,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "market_caps", force: :cascade do |t|
```

Comment:
Let's stash these lines for this particular PR please.

Reply:
Oops sorry, will clean that up.